README.undock.md

```diff
@@ -1,41 +1,48 @@
 # Experimental undock branch

```

Comment:
Can we delete the PS script now?

Reply:
It's not on parity yet, but as soon as it is yes, that's the plan.